src/tests/ftest/util/soak_utils.py

```diff
@@ -389,23 +389,20 @@ def wait_for_pool_rebuild(self, pool, name):
 
     """
     rebuild_status = False
-    self.log.info(
-        "<<Wait for %s rebuild on %s>> at %s", name, pool.uuid, time.ctime())
+    self.log.info("<<Wait for %s rebuild on %s>> at %s", name, pool.uuid, time.ctime())
     try:
         # # Wait for rebuild to start
-        # pool.wait_for_rebuild(True)
+        # pool.wait_for_rebuild_to_start()
         # Wait for rebuild to complete
-        pool.wait_for_rebuild(False)
+        pool.wait_for_rebuild_to_stop()
```

Comment:
This method does not exist; should it be pool.wait_for_rebuild_to_end() ?

Reply:
Fixed